src/Microsoft.DotNet.Cli.Utils/RuntimeConfig.cs

```diff
@@ -1,63 +0,0 @@
-// Copyright (c) .NET Foundation and contributors. All rights reserved. 
-// Licensed under the MIT license. See LICENSE file in the project root for full license information. 
-
-using Newtonsoft.Json;
```

Comment:
Need to move the type since util is netstandard (some part is used in full framework)